src/app/leaves/components/LeaveActionRender.tsx

```diff
@@ -75,7 +81,16 @@ const LeaveTableActionRender = ({ leaveRequestDetails }: Props) => {
       {
         title: "Edit",
         id: 2,
-        onClick: handleEditInfo,
+        onClick: () => {
+          if (requestId !== undefined) {
```

Comment:
Why would that be the case? When we won't have the id?

Reply:
Sir @Monfernape Actually, I made the type optional, which is why I added this check. However, I've now fixed the issue by removing the optional type.

Reply:
Let's remove this check then.